tm2/pkg/p2p/test_util.go

```diff
@@ -19,24 +19,6 @@ const testCh = 0x01
 
 // ------------------------------------------------
 
-func AddPeerToSwitchPeerSet(sw *Switch, peer Peer) {
```

Comment:
Why was this removed?

Reply:
I realized it was unused, but not caught by the `unused` linter 🤷‍♂️